hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/ConfigurationKeys.java

```diff
@@ -128,7 +128,8 @@ public static String accountProperty(String property, String account) {
   public static final String FS_AZURE_ENABLE_DELEGATION_TOKEN = "fs.azure.enable.delegation.token";
   public static final String FS_AZURE_DELEGATION_TOKEN_PROVIDER_TYPE = "fs.azure.delegation.token.provider.type";
 
-  public static final String ABFS_EXTERNAL_AUTHORIZATION_CLASS = "abfs.external.authorization.class";
+  /** External Authorizer **/
+  public static final String FS_AZURE_ABFS_AUTHORIZER = "fs.azure.abfs.authorizer";
```

Comment:
Now, this is going to be backwards incompatible. Which happens in the code anyway. Is everyone happy with this?

Reply:
There were no consumers for abfs.external.authorization.class.